File: Beecrowd/1113 Crescente e Decrescente/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int X, Y;

        while(true){
            X = sc.nextInt();
            Y = sc.nextInt();
            if(X == Y)
                break;
            else if(X > Y)
                System.out.println("Decrescente");
            else
                System.out.println("Crescente");  
        }

        sc.close();
    }
}

File: Beecrowd/1113 Crescente e Decrescente/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	int X, Y;
	
	do{
	cin >> X >> Y;
	
	if(X > Y)
		cout << "Decrescente" << endl;
	else if (X < Y)
		cout << "Crescente" << endl;
	}while(X != Y);
 
    return 0;
}